study.md

```diff
@@ -100,7 +108,11 @@ pairs.
 | occupation | 'propane and propane accessories salesman' |
 
 ```javascript
-// your answer here
+const hank = {
+  family_name: 'hill',
+  given-name: 'hank',
```

Comment:
Key names with special characters need to be wrapped in quotation marks.  `given-name` will be interpreted as the difference between the two variables `given` and `name`.